api/pacman-api-compliance/src/main/java/com/tmobile/pacman/api/compliance/controller/FilterController.java

```diff
@@ -407,4 +412,15 @@ public ResponseEntity<Object> getPolicyComplianceFilter(@RequestBody FilterReque
         }
         return ResponseUtils.buildSucessResponse(response.getRes());
     }
+    @PostMapping(path = "/v1/filters/eventdate")
```

Comment:
add empty line - keep code consistent. If you are not sure how it should look - just look abound in the file and apply the same format principles.

Reply:
done